scripts/Inductors/Inductor_generic/Readme.md

```diff
@@ -0,0 +1,3 @@
+Generic generator for inductor footprints. Use it to exactly duplicate the suggested footprint given in the datasheet (as oposed to the IPC generators which do not rely on the suggested footprint). 
+
+It supports generating footprints for inductors with rectangular base and possibly some corners chamfered, rounded or cut out. (Example TDK SLF6028)
```

Comment:
This sentence is rather awkward. Could change `...`with rectangular base and possibly some corners...` to `...with a rectangular base and corners which are...` or something else you prefer.

Reply:
Fixed but had a better idea than your suggestion